input/docs/en/matlab_start.md

```diff
@@ -36,15 +36,14 @@ Notice dark color in blue channel, which is not part of yellow color unlike red
 Set non-color channels to zero 
 
 ``` matlab
-zeroMatrix=zeros(size(A,1),size(A,2));
-R1=cat(3,R,zeroMatrix,zeroMatrix); 
-G1=cat(3,zeroMatrix,G,zeroMatrix);
-B1=cat(3,zeroMatrix,zeroMatrix,B);
+R(:,:,[2,3])=0; 
+G(:,:,[1,3])=0;
+B(:,:,[1,2])=0;
```

Comment:
Using `R`, `G` and `B`  causes unexpected results. Matlab doesn't warn you, but rewrites whole variables. It is 0 everywhere and `B` variable has size in 3rd dimension only 2. You need to use new variables (`R1`,...) and copy original RGB image in there. (keep creating of these three copies on one line for better readability).